mappings/net/minecraft/fluid/FlowableFluid.mapping

```diff
@@ -48,12 +48,14 @@ CLASS net/minecraft/class_3609 net/minecraft/fluid/FlowableFluid
 		ARG 4 flowDirection
 		ARG 5 flowTo
 		ARG 6 flowToBlockState
+		ARG 7 fluidState
+		ARG 8 fluid
 	METHOD method_15739 getLevelDecreasePerBlock (Lnet/minecraft/class_4538;)I
 		ARG 1 world
 	METHOD method_15740 (Lnet/minecraft/class_4538;Lnet/minecraft/class_2338;)I
 		ARG 1 world
 		ARG 2 pos
-	METHOD method_15741 (Lnet/minecraft/class_3610;)I
+	METHOD method_15741 getBlockStateLevel (Lnet/minecraft/class_3610;)I
```

Comment:
```suggestion
	METHOD method_15741 getLevelOf (Lnet/minecraft/class_3610;)I
```

Reply:
This method is called in the `toBlockState` method of WaterFluid and LavaFluid, to set the "level" block state

Reply:
How is this a setter when its descriptor is `int method_15741(BlockState)`?

Reply:
I meant this method is used to set "level" block state _in_ `toBlockState`

Reply:
Oh, 3610 is fluid state